filmshelf_client/src/components/home/home.css

```diff
@@ -33,55 +33,117 @@ body {
   padding: 20px;
 }
 
-/*Featured Item Section */
-.featured-items {
+/* Swiper styling */
+.popular-carousel {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+/* Position No.# on top of the white card */
+.swiper-slide-number {
+  position: absolute;
+  bottom: 10px;  /* Adjust position to be above the card */
+  right: 80px;  /* Align the number slightly to the left */
+  color: #000000;
+  font-size: 18px;
+  z-index: 20;  /* Ensure it appears above the card */
+}
+
+/* Custom navigation for arrows placed next to the No.# */
+.custom-navigation {
+  position: absolute;
+  bottom: 230px; /* Align with the No.# */
```

Comment:
lets try to have the arrows inside of the containers instead of relative to the screen.